packages/core/cli.ts

```diff
@@ -74,6 +74,15 @@ yargs(process.argv.slice(2))
 				})
 				.hide("fuzzFunction")
 
+				.option("id_sync_file", {
+					describe:
+						"File used to sync edge ID generation. " +
+						"Needed when fuzzing in multi-process modes",
```

Comment:
Should the fork mode automatically be activated if `id_sync_file` is set? If not we should mention the necessary libFuzzer flags and the default that is used if no sync file is set.

Reply:
Ah, good point, didn't think about that.

Reply:
Still, should we mention the affected libFuzzer modes or, if the flag should only be used internally, hide it?

Reply:
I think hiding the flag would be the better option for now.

Reply:
Done!